docs/releases/1.10.txt

```diff
@@ -55,6 +55,31 @@ Minor features
 :mod:`django.contrib.auth`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Support for the Argon2_ password hash has been added.  Argon2 is the winner
+  of the recent `Password Hashing Competition`_, which has been organized
+  by the community in similar style as the NIST competitions that selected
+  the hash SHA-3 and cipher AES.
```

Comment:
I’m not afraid to oversell it, I’m just not partial to name dropping of unrelated technologies.  But it’s up to the Django folks to make that call; it sure is illustrative.